<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>portes</class>
 <widget class="QDialog" name="portes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView_port">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>471</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>130</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NUM</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>180</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CODE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>230</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TEMP OUVERT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>270</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TEMP FERME</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_NUMPORT">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>130</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_CODEPORT">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>180</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_TEMPOUVERT">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>230</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_TEMPFERME">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>270</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Ajouter">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>AJOUTER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Afficher">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>490</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Afficher</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_supprimer">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>500</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_TRI">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Tri</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Recherche : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_search">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
